scripts/verify-replace.sh

```diff
@@ -16,26 +20,57 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-error.XXX)
+    STDOUTFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-output.XXX)
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
     if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+        ERRORREPOLIST+="$(basename ${repo}) "
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies 2> ${ERRFILE} 1> ${STDOUTFILE}
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+    ERRORREPOLIST+="$(basename ${repo}) " 
+    ERRORFILELIST+="${ERRFILE}  "
+    STDOUTFILELIST="${STDOUTFILE} "
```

Comment:
isn't it missing the plus `+`?
```suggestion
    STDOUTFILELIST+="${STDOUTFILE} "
```

Reply:
ideally yes, i just didn't see it and don't know how it worked without the plus sign, probably i tested wrong.
changed it now